### PR TITLE
enable connectivity checks

### PR DESCRIPTION
**Pull request by sanchezl**

### CODE REVIEW

File: pkg/operator/connectivitycheckcontroller/connectivity_check_controller.go [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+			TargetEndpoint: address,
+		},
+	}
+	for _, option := range options {
+		option(check)
+	}
+	return check
+}
+
+func withTlsClientCert(secretName string) func(*v1alpha1.PodNetworkConnectivityCheck) {
```
Comment (deads2k) [inline pkg/operator/connectivitycheckcontroller/connectivity_check_controller.go:246]: nit: we use TLS

**Comment by deads2k** [comment]
/lgtm

I think I looked at this before.

File: pkg/operator/connectivitycheckcontroller/connectivity_check_controller.go (hunk 1 above)
Comment (sanchezl) [reply pkg/operator/connectivitycheckcontroller/connectivity_check_controller.go:246]: @deads2k  https://github.com/openshift/cluster-openshift-apiserver-operator/pull/368

**Pull request merged**